apps/myaccount/src/components/multi-factor-authentication/multi-factor-authentication.tsx

```diff
@@ -198,27 +198,35 @@ export const MultiFactorAuthentication: React.FunctionComponent<MfaProps> = (pro
                                 isSuperTenantLogin={ isSuperTenantLogin() }
                                 onEnabledAuthenticatorsUpdated={ handleEnabledAuthenticatorsUpdated }
                                 triggerBackupCodesFlow={ () => setInitBackupCodeFlow(true) }
-                                handleSessionTerminationModalVisibility={ 
-                                    () => setShowSessionTerminationModal(true) 
-                                }
+                                handleSessionTerminationModalVisibility={ () => {
+                                    // Show the session termination modal only if the backup code flow is not activated
+                                    // to stop showing duplicate modals.
+                                    if (isSuperTenantLogin() && isTOTPEnabled && isBackupCodesConfigured) {
```

Comment:
can't we directly use `isSuperTenantLogin` instead of `isSuperTenantLogin()`
https://github.com/wso2/identity-apps/pull/3525/files#diff-8c5dd713351a819b1675157236d227d505a48607facfdd61b56aee36299d9c7fR198

Reply:
`isSuperTenantLogin` is a function defined in the component. In here we are using `isSuperTenantLogin()` to call that function and get the output rather than passing the function reference